index.js

```diff
@@ -420,8 +420,27 @@ module.exports = {
 			'error',
 			'never',
 		],
-		// Disabled because of https://github.com/xojs/eslint-config-xo/issues/27
-		// 'object-property-newline': 'error',
+		'object-curly-newline': [
+			'error',
+			{
+				ObjectExpression: {
+					multiline: true,
+					minProperties: 2,
+					consistent: true,
+				},
+				ObjectPattern: 'never',
```

Comment:
It makes no sense to completely disallow putting properties on multiple line here.